airbyte-webapp/src/components/JobItem/components/MainInfo.tsx

```diff
@@ -68,12 +68,12 @@ const MainInfo: React.FC<MainInfoProps> = ({ job, attempts = [], isOpen, onExpan
     } else if (isPartialSuccess) {
       status = "partialSuccess";
     } else {
-      return <FormattedMessage id="sources.jobStatus.unknown" />;
+      return <FormattedMessage id="jobs.jobStatus.unknown" />;
```

Comment:
:information_source: I've renamed this prefix to `jobs`, since this can show for source, connections, destinations, so I felt `jobs` might be a better prefix.